src/components/form/components/input/numeric_input.tsx

```diff
@@ -14,17 +14,17 @@ interface NumericInputProps
 
 export function NumericInput(props: NumericInputProps) {
   const { label, required, ...rest } = props;
-  const field = useFieldContext<number | undefined>();
+  const field = useFieldContext<string | undefined | null>();
   const id = useFieldId(field.name);
   const error = useErrors(field);
   const intent = getIntent(error);
 
-  function onChange(value: number, valueAsString: string) {
+  function onChange(_: number, valueAsString: string) {
     if (valueAsString === '') {
-      return field.handleChange(undefined);
+      return field.handleChange(null);
```

Comment:
Is passing `undefined` what reverts the value to its default value?

Reply:
Yes, then if you want nothing in the input like you wanted before, we have to set "null"

Reply:
Why don't you pass an empty string then? The validation can transform an empty string to `undefined` then. The implementation of the NumericInput form component should stay close to a normal numeric input and not expect the `null` value.